.github/workflows/ci.yml

```diff
@@ -0,0 +1,42 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
```

Comment:
Note - the git diff is showing changes (like the addition of this file) that are already in main. This is purely a visual issue, could be fixed by a rebase but that can get quite messy and result in lost work/chronology of commits.